src/pybind/mgr/rbd_support/schedule.py

```diff
@@ -243,7 +244,7 @@ def from_string(cls, start_time):
             return None
 
         try:
-            t = time.fromisoformat(start_time)
+            t = parse(start_time).timetz()
```

Comment:
Nit: can we use `isoparse` instead?

Reply:
`isoparse` don't parse strings without date. It always expect date comes first and then optionally time. I think something like this would work:
```
t = isoparse('1970-01-01 ' + start_time).timetz()
```
though it does not look very nice...

Reply:
Ack -- I'm fine w/ it as-is then.